### PR TITLE
AboutUsView

### LINKED ISSUE #1407: Create "About us" view
Will be great if you will create some beautiful page with our pretty faces and links to the github projects/accounts

### PR DESCRIPTION
**Pull request by sanyavertolet**
This PR closes #1407 

### What's done:
 * Added `AboutUsView`
 * Renamed `About` -> `About us`

### How it looks:

https://user-images.githubusercontent.com/35039155/199695566-fca339a7-d6d6-4ca7-8ca3-34a1260cb517.mov

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                    }
+                }
+            }
+        }
+    }
+
+    companion object :
+        RStatics<AboutUsViewProps, AboutUsViewState, AboutUsView, Context<RequestStatusContext>>(AboutUsView::class) {
+        private const val DEFAULT_AVATAR_SIZE = "175"
+        private const val GITHUB_AVATAR_LINK = "https://avatars.githubusercontent.com/"
```
Comment (Cheshiriks) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt:167]: Maybe take our avatars from the Save, and not GitHub?

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt:167]: Not everyone has his face on saveourtool, but everybody has it on GitHub

Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt:167]: GitHub, definitely

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+            +"About us"
+        }
+    }
+
+    private fun ChildrenBuilder.renderSaveourtoolInfo() {
+        div {
+            div {
+                className = ClassName("mt-3 d-flex justify-content-center align-items-center")
+                div {
+                    className = ClassName("col-6")
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt:74]: can we render it so all cards will have a common alignment? (не выпирают углы)

image

**Review by akuleshov7 — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt (hunk 2 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt]: Done
screenshot

**Pull request merged**